### PR TITLE
internal/v4: implement charm actions & tests

### PR DESCRIPTION
**Pull request by urosj**

### CODE REVIEW

File: internal/v4/api.go [hunk 1]
```diff
@@ -145,7 +145,15 @@ func (h *handler) metaManifest(getter router.ItemGetter, id *charm.URL, path str
 // GET id/meta/charm-actions
 // http://tinyurl.com/kfd2h34
 func (h *handler) metaCharmActions(getter router.ItemGetter, id *charm.URL, path string, flags url.Values) (interface{}, error) {
-	return nil, errNotImplemented
+	//return nil, errNotImplemented
```
Comment (frankban) [inline internal/v4/api.go:148]: It seems we can safely remove this comment now.

**Comment by frankban** [comment]
:+1: with the changes we discussed:
- include the AssertJSON fix from Roger;
- fix the include test to actually request charm-actions;
- use a charm with some actions.

Thank you!

**Comment by jujugui** [comment]
please test this

**Comment by bac** [comment]
This PR is causing Jenkins to break since there is no source repository.  I'm closing it.

**Pull request closed without merging**